### ISSUE TITLE
AltServer could not find this Device

### ISSUE THREAD

**Issue by StiviiK** [rileytestut/AltStore#99]
After installing `AltServer` on my Mac and installing the `AltStore` on my iPhone I can only install/refresh Apps while my iPhone is connected via USB, per WiFi it isn't working and errors out with the error from above.

![Screenshot of the Error](https://user-images.githubusercontent.com/8115642/74357092-a9bc3e80-4dbf-11ea-8349-b212b427ebd1.jpeg)

**Comment by BluePsychoRanger** [comment]
I had this same issue. I think I figured out how to fix it cuz it just recently started happening after I updated my iPhone to iOS 13.3. When I connected my phone to my MacBook, I had to click "Trust this Computer" again even tho I had done it before.

I also synced my iPhone and checked the following boxes: "Show this iPhone when on WiFi" and "Automatically sync when this iPhone is connected." 

Previously my iPhone wasn't showing up in the sidebar of Finder, and I got this error, but once I did those steps, I disconnected my iPhone from my MacBook then tried refreshing and it worked first try.
